rapid7_vulndb/komand_rapid7_vulndb/util/utils.py

```diff
@@ -0,0 +1,311 @@
+from komand.exceptions import PluginException
+
+from typing import Dict, List, Callable, Tuple
+import requests
+import copy
+import sys
+from urllib.parse import urljoin
+
+
+class R7VDB:
+    """
+    Rapid7 vulnerability database API utilities
+    This class is used by the rapid7_vulndb plugin for
+    extracting https://vdb.rapid7.com/swagger_doc API data
+
+    Two main functions that are used by the plugin are
+        1. paginate_search
+        2. make_content_result
+
+    most of the methods implemented as class methods to avoid initialization
+    """
+
+    search_url: str = "https://vdb-kasf1i23nr1kl2j4.rapid7.com/v1/search"
```

Comment:
Might be worth having these share the same root domain (along with info endpoint in connection test) to avoid potential fat fingering in future if new endpoints are added.